core/src/main/java/io/seata/core/rpc/netty/RmNettyRemotingClient.java

```diff
@@ -82,7 +82,8 @@ public void init() {
             if (resourceManager != null
                     && !resourceManager.getManagedResources().isEmpty()
                     && StringUtils.isNotBlank(transactionServiceGroup)) {
-                getClientChannelManager().reconnect(transactionServiceGroup);
+                boolean failFast = NettyClientConfig.isEnableClientChannelCheckFailFast();
```

Comment:
I understand that fastfail should be an attribute on the application side, not a netty attribute.

Reply:
done